beacon/beacon-basic-blob.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>W3C Beacon Basic Blob Test</title>
+    <meta name="timeout" content="long">
+    <script src="../resources/testharness.js"></script>
+    <script src="../resources/testharnessreport.js"></script>
```

Comment:
In all these files, the `resources` URLs in those script elements need to be changed to an absolute path:

```html
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
```